### PR TITLE
[v2.3.0] Bump Plugin Version and Update Readme Files

### PR DESCRIPTION
**Pull request by Vedant-Gandhi**
### Changes 
- Updated the Plugin Version to 2.3.0
- Updated the po and mo files.
- Updated Readme files reflecting the changelog and plugin version.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -127,6 +127,16 @@ Please post your problem in [our free support forum](https://github.com/rtCamp/n
```
Comment (Pathan-Amaankhan) [inline readme.txt:127]: Update the `Stable tag` also to the latest version i.e. `2.3.0` in this file.

File: README.md [hunk 2]
```diff
@@ -139,6 +139,15 @@ Please post your problem in [our free support forum](https://github.com/rtCamp/n
```
Comment (Pathan-Amaankhan) [inline README.md:139]: Update the Stable tag also to the latest version i.e. `2.3.0` in this file.

**Review by Pathan-Amaankhan — CHANGES_REQUESTED** [review]

File: readme.txt [hunk 3]
```diff
@@ -6,7 +6,7 @@ License: GPLv2 or later (of-course)
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 Requires at least: 3.0
 Tested up to: 6.1
```
Comment (Pathan-Amaankhan) [inline readme.txt:8]: Missed updating Tested up to. Please makesure that fields in readme.md and readme.txt are similar.

**Review by Pathan-Amaankhan — CHANGES_REQUESTED** [review]

**Review by Pathan-Amaankhan — APPROVED** [review]
LGTM

**Review by Pathan-Amaankhan — CHANGES_REQUESTED** [review]
Please raise a develop PR first before merging to master.
We will first test the changes on develop.

Next, if we have already merged the fixes/features on master, we can directly raise a version bump PR for master.

If the changes are not merged, then a master <-> develop sync PR should be raised to migrate the develop commits to master after testing develop. (ref PR: https://github.com/rtCamp/nginx-helper/pull/318)

File: README.md [hunk 4]
```diff
@@ -113,6 +113,18 @@ define( 'RT_WP_NGINX_HELPER_REDIS_PORT', '6000' );
 define( 'RT_WP_NGINX_HELPER_REDIS_PREFIX', 'page-cache:' );
 ```
 
+**Q. Can I override the redis socket path, username, password?**
```
[the remark this answers is not in the document]
Comment (Pathan-Amaankhan) [reply README.md:116]: Have we added this on develop?
If no than please raise a PR that adds the same on develop...

[the remark this answers is not in the document]
Comment (Vedant-Gandhi) [reply README.md:116]: Okay will add it.

[the remark this answers is not in the document]
Comment (Vedant-Gandhi) [reply README.md:116]: Ref - https://github.com/rtCamp/nginx-helper/pull/375

**Review by Pathan-Amaankhan — APPROVED** [review]

**Pull request merged**